php/helpers.php

```diff
@@ -253,20 +253,24 @@ function ArrayToHTML($array){
 
 	$str = "<table style='border: solid 1px'>";
 
+	$columnNames = array();
+
 	foreach($array as $id => $row){
 		$str .= "<tr style='border: solid 2px'><td>#</td>";
 		foreach($row as $key => $value){
 			$str .= "<th style='border: solid 2px'>";
 			$str .= "$key";
 			$str .= "</th>";
+			array_push($columnNames, $key);
 		}
 		$str .= "</tr>";
 		break;
 	}
 
 	foreach($array as $id => $row){
 		$str .= "<tr style='border: solid 1px'><td>$id</td>";
-		foreach($row as $key => $value){
+		foreach($columnNames as $columnName) {
+			$value = $row[$columnName];
```

Comment:
Does this need an isset check? it may throw a huge number of warnings for missing values without the check. Try uncommenting lines 2-4 in index.php to see if this throws loads of warnings or not

Reply:
It does!